modules/programs/firefox.nix

```diff
@@ -10,13 +10,15 @@ let
 
   jsonFormat = pkgs.formats.json { };
 
-  mozillaConfigPath =
```

Comment:
The `mozillaConfigPath` variable is now used for the `native-messaging-hosts` directory.

Looking at https://librewolf.net/docs/faq/#how-do-i-get-native-messaging-to-work it seems `~/.librewolf/native-messaging-hosts` is the path for LibreWolf. So perhaps could have `mozillaConfigPath = "~/.librewolf"` when `isLibrewolf == true`?